### PR TITLE
Implement Log Persistency

### PR DESCRIPTION
**Pull request by hnicke**
Resolves: #83
- Implement PersistentAppender which writes logs to given logfile
- Change Log object: Upon construction, it parses logs from disk
- Add tests for the new functionality

### CODE REVIEW

File: core/src/test/java/org/opentosca/toscana/core/csar/CsarServiceImplTest.java [hunk 1]
```diff
@@ -37,7 +39,8 @@ public void submitCsar() throws Exception {
     @Test
     public void deleteCsar() throws Exception {
         File csarDir = new File(tmpdir, identifier);
-        Csar csar = new CsarImpl(identifier);
+        // TODO
```
Comment (c-mueller) [inline core/src/test/java/org/opentosca/toscana/core/csar/CsarServiceImplTest.java]: Still todo?

File: core/src/test/java/org/opentosca/toscana/core/dummy/DummyTransformationService.java [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-import org.opentosca.toscana.core.transformation.Transformation;
-import org.opentosca.toscana.core.transformation.TransformationService;
-import org.opentosca.toscana.core.transformation.platform.Platform;
+    import org.opentosca.toscana.core.csar.Csar;
+    import org.opentosca.toscana.core.transformation.Transformation;
+    import org.opentosca.toscana.core.transformation.TransformationService;
+    import org.opentosca.toscana.core.transformation.platform.Platform;
 
-//@Service //TODO If Transformation Service has been implemented
+//@Service //TODO If Transformation Service has been implemented
```
Comment (c-mueller) [inline core/src/test/java/org/opentosca/toscana/core/dummy/DummyTransformationService.java]: probably a "merge fail" this comment can be removed

File: core/src/test/java/org/opentosca/toscana/core/csar/CsarServiceImplTest.java (hunk 1 above)
Comment (hnicke) [reply core/src/test/java/org/opentosca/toscana/core/csar/CsarServiceImplTest.java]: fixed

File: core/src/test/java/org/opentosca/toscana/core/dummy/DummyTransformationService.java (hunk 2 above)
Comment (hnicke) [reply core/src/test/java/org/opentosca/toscana/core/dummy/DummyTransformationService.java]: fixed

**Pull request merged**